### DISCUSSION TITLE
tabs apperance

### DISCUSSION THREAD

**Discussion by zibertszn** [kovidgoyal/kitty#4798] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/72556979/157105590-790fbd8e-51c5-4f51-8085-312695ca2ecc.png)
i wanna tabs like this, how can i do it

Now i have this
![image](https://user-images.githubusercontent.com/72556979/157105690-6a1a5000-4403-4229-adb7-1a18e2deb6e7.png)

**Comment by page-down** [comment, marked as answer]
~/.config/kitty/kitty.conf
```conf
shell_integration no-title
```

As for how to customize the title
https://wiki.archlinux.org/title/zsh#xterm_title
